### ISSUE TITLE
UFW/GUFW refuses to install/update

### ISSUE THREAD

**Issue by Diolinux** [pop-os/pop#1992]
NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os
 
 
 
The packge with a problem is "ufw", the "uncomplicated firewall"

**Issue/Bug Description:**

The app itself doest seems to have a problem, but dpkg aren't able to install/updated it.

**Steps to reproduce (if you know):**

Just try to install our update the software, like:

`sudo apt install ufw
`
**Expected behavior:**

A preparar para desempacotar .../ufw_0.36-6ubuntu1_all.deb ...
A descompactar ufw (0.36-6ubuntu1) ...
Configurando ufw (0.36-6ubuntu1) ...
dpkg: erro ao processar o pacote ufw (--configure):
 o subprocesso instalado, do pacote ufw, o script post-installation retornou erro do status de saída 10
A processar 'triggers' para man-db (2.9.1-1) ...
A processar 'triggers' para rsyslog (8.2001.0-1ubuntu1.1) ...
A processar 'triggers' para systemd (245.4-4ubuntu3.13pop0~1631829301~20.04~a958ca9) ...
Erros foram encontrados durante o processamento de:
 ufw
E: Sub-process /usr/bin/dpkg returned an error code (1)

**Comment by ericrodriguesfer** [comment]
Same ocurrency in ubuntu 20.04 LTS

**Comment by leviport** [comment]
I don't think this is a problem anymore, but I can reopen this if I'm mistaken.
